### ISSUE TITLE
Top site title is not easily readable 

### ISSUE THREAD

**Issue by vesta0** [mozilla-mobile/fenix#15123]
## User Story
- As a user, I want to be able to easily read a top site title in both dark and light mode, so I can easily and effortlessly find what I am looking for. 

### Acceptance Criteria
- There contrast is high enough between the top site title font colour and homescreen colour (light/dark) making the title easily readable.
- The font type/size of the top site title is optimized for readability.

**Comment by yoasif** [comment]
Dupe of https://github.com/mozilla-mobile/fenix/issues/13850

**Comment by brampitoyo** [comment]
The solution is to increase the colour contrast value for _all_ captions and secondary texts. When done, it will not only make Top Site labels more readable – all other small texts will follow.

| Before | After |
| --- | --- |
| <img width="360" src="https://user-images.githubusercontent.com/51007/93394258-0d639880-f8c8-11ea-8447-b7610835b49d.png"> | <img width="360" src="https://user-images.githubusercontent.com/51007/93394316-266c4980-f8c8-11ea-8a85-d29ea7550f96.png"> |
| Light Grey 90 `#80808e` | Dark Grey 10 `#52525e` |
| <img width="360" src="https://user-images.githubusercontent.com/51007/93394003-97f7c800-f8c7-11ea-871b-d12a2c592573.png"> | <img width="360" src="https://user-images.githubusercontent.com/51007/93394130-d68d8280-f8c7-11ea-8462-ea4f187570d4.png">
| Light Grey 70 `#9f9fad` | Light Grey 60 `#afafbb` |

These colour values, which should be applied to all captions and secondary texts, have sufficient contrast ratio to pass WCAG 2.0 AAA standard. Additionally, the new Dark Theme colour value should also be used for Private Theme.

**Comment by vesta0** [comment]
Thank you so much @brampitoyo :)

**Comment by sflorean** [comment]
Verified as fixed on Nightly 9/21 with Samsung Galaxy Note10 (Android 10).

**Comment by pellaeon** [comment]
It still look unnecessarily small on my device (Samsung Galaxy A3 2017 (A320), Android 9, Firefox Nightly 11/20):
![Screenshot_20201121-133830_Firefox_Nightly](https://user-images.githubusercontent.com/916913/99884860-302f6780-2bff-11eb-8a4e-1dac6fd9b05f.png)

A lot of unnecessary margin around them too. It looks quite different from @brampitoyo 's screenshot.
